code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2512,16 +2511,27 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	name = "Beef Fizz"
 	description = "This is beef fizz, BEEF FIZZ, THERE IS NO GOD"
 	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_BOTANY | CHEMICAL_GOAL_BARTENDER_SERVING
-	boozepwr = 15
-	quality = DRINK_BAD
+	boozepwr = 0
```

Comment:
This should have at least some power

Reply:
It lacks of alcoholic components, how are you supposed to get drunk off of beef, ice and lemon juice?

Reply:
Then it shouldn't be under the alcohol subcategory